vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/dto/BulkVersionRequest.java

```diff
@@ -0,0 +1,34 @@
+package ca.on.oicr.gsi.vidarr.server.dto;
+
+import java.util.List;
+
+public final class BulkVersionRequest {
+
+  String newVersionKey;
+  String oldVersionKey;
```

Comment:
Would it make sense to rename these `newProviderVersion`/`oldProviderVersion`? I think that would be much clearer if so

Reply:
These aren't providers though. Or, at least they won't be by the time we do the rearrangement of the providers...

Right now, we've got a LIMS key object that looks like this:
```
{
  "provider": "pinery-miso-v7",
  "identifier": "123_1_LDI4567",
  "version": "0123456789abcedf",
  "timestamp": "2020-01-25T12:13:14Z"
}
```

the goal is to rewrite them to look like this:
```
{
  "provider": "pinery-miso",
  "identifier": "123_1_LDI4567",
  "versions": {
    "pinery-hash-v7": "0123456789abcedf",
    "pinery-modified": "2020-01-25T12:13:14Z"
  }
}
```
As part of the migration of data from Niassa to Vidarr, we're going to have to do that rewrite. So, the two `VersionKey` fields would be `"pinery-hash-v7"`, not the provider.

Reply:
Ah, I get it now. Thanks